storage/server/backend.go

```diff
@@ -9,6 +9,7 @@ import (
 	"log"
 
 	pb "github.com/opiproject/opi-api/storage/v1/gen/go"
+	emptypb "google.golang.org/protobuf/types/known/emptypb"
```

Comment:
@PWAlessi is that how that is actually done in golang ?

Reply:
You could probably just alias it to empty.